capabilities.json

```diff
@@ -149,44 +121,70 @@
-                    "placeHolderText": "Auto",
                     "type": {
                         "numeric": true
                     },
                     "suppressFormatPainterCopy": true
                 },
-              "fontSize": {
-                "displayName": "Text Size",
-                "displayNameKey": "Visual_TextSize",
-                "type": {
-                  "formatting": {
-                    "fontSize": true
-                  }
-                }
-              },
-               "orientation": {
-                    "displayName": "Orientation",
-                    "displayNameKey": "Visual_Orientation",
+                "fontSize": {
                     "type": {
-                      "enumeration": [
-                        {
-                          "value": "Horizontal",
-                          "displayName": "Horizontal",
-                          "displayNameKey": "Visual_Horizontal"
-                        },                       
-                        {
-                          "value": "Vertical",
-                          "displayName": "Vertical",
-                          "displayNameKey": "Visual_Vertical"
+                        "formatting": {
+                            "fontSize": true
+                        }
+                    }
+                },
+                "fontFamily": {
+                    "type": {
+                        "formatting": {
+                            "fontFamily": true
                         }
-                      ]
                     }
-                  }
+                },
+                "bold": {
+                    "type": {
+                        "bool": true
+                    }
+                },
+                "italic": {
+                    "type": {
+                        "bool": true
+                    }
+                },
+                "underline": {
+                    "type": {
+                        "bool": true
+                    }
+                },
+                "orientation": {
+                    "type": {
+                        "enumeration": [
+                            {
+                                "value": "Horizontal",
+                                "displayName": "Horizontal",
+                                "displayNameKey": "Visual_Horizontal"
+                            },
+                            {
+                                "value": "Vertical",
+                                "displayName": "Vertical",
+                                "displayNameKey": "Visual_Vertical"
+                            }
+                        ]
+                    }
+                }
             }
         }
-    }
-}
+    },
+    "tooltips": {
+        "supportedTypes": {
+            "default": true,
+            "canvas": true
+        },
+        "roles": [
+            "tooltips"
+        ]
+    },
+    "supportsHighlight": true,
```

Comment:
We have to remove `supportsHighlight` support for now, since it will break current users expectations. And there is no a mechanism for now to set default interaction behavior.

Reply:
Removed `supportsHighlight`.